spec/API_specification/linear_algebra_functions.md

```diff
@@ -275,9 +275,35 @@ TODO
 TODO
 
 (function-svd)=
-### svd()
+### svd(x, /, *, compute_uv=True, full_matrices=True)
 
-TODO
+Computes the singular value decomposition `A = USV` of a matrix (or stack of matrices) `x`.
+
+#### Parameters
+
+-   **x**: _&lt;array&gt;_
+
+    -   input array having shape `(..., M, N)` and whose innermost two dimensions form matrices on which to perform singular value decomposition. Must have a data type of either `float32` or `float64`.
+
+-   **compute_uv**: _bool_
+
+    -   If `True`, compute the left and right singular vectors and return as `u` and `v`, respectively. Default: `True`.
+
+-   **full_matrices**: _bool_
+
+    -   If `True`, compute full-sized `u` and `v`, such that `u` has shape `(..., M, M)` and `v` has shape `(..., N, N)`. If `False`, compute on the leading `K` singular vectors, such that `u` has shape `(..., M, K)` and `v` has shape `(..., N, K)` and where `K = min(M, N)`. This option must be ignored if `compute_uv` is `False`. Default: `True`.
```

Comment:
In NumPy, `v.shape == (..., K, N)`, not `(..., N, K)`. I suppose we wanna follow NumPy here?

Reply:
Indeed. I updated below, but not here. PR should be fixed.